### ISSUE TITLE
Android does not receive push notifications

### ISSUE THREAD

**Issue by justinemaka** [Pushwoosh/pushwoosh-flutter#73]
Hi,

After setting up pushwoosh on my flutter application with firebase, my ios apps receive notifications, but android doesn't receive notifications.

I've followed the setup on [flutter integration docs](https://docs.pushwoosh.com/platform-docs/pushwoosh-sdk/cross-platform-frameworks/flutter) and even registered to listen to notifications with: `Pushwoosh.getInstance.registerForPushNotifications()`

Can I have any help with this?

**Comment by enginseer-dev** [comment]
Hello, @justinemaka 

It sounds like there might be an integration issue specific to the Android setup.
We would appreciate if you could send us the device logs, so we can investigate the issue.
You can email the logs to help@pushwoosh.com

Please record the logs beginning with the app installation and continue until the point where the notification is expected to be received.

**Comment by justinemaka** [comment]
I've been able to resolve the issue.
I had integrated firebase messaging and pushwoosh. Notifications from firebase was showing up, but those from pushwoosh weren't coming through.

I later found out from [this comment](https://github.com/Pushwoosh/pushwoosh-phonegap-plugin/issues/351#issuecomment-871296873) which informed me to  create a router service to distribute events between the services.

PS: Follow this guide on [Pushwoosh android faq guide](https://docs.pushwoosh.com/platform-docs/pushwoosh-sdk/android-push-notifications/android-faq#using-pushwoosh-sdk-with-other-fcm-services), or [this comment](https://github.com/Pushwoosh/pushwoosh-phonegap-plugin/issues/351#issuecomment-871296873) to properly implement this solution.
